sdk/react/src/BottomOverlay/BottomOverlay.js

```diff
@@ -0,0 +1,670 @@
+// @flow
```

Comment:
We lost git history for this file :(

Reply:
Unfortunately yes, initially I just moved and committed the file, and only after that changed it, but Git lost it anyway, don't know why :(